polaris-react/src/components/IndexFilters/IndexFilters.scss

```diff
@@ -30,22 +30,17 @@ $spinner-size: 20px;
 
 .TabsWrapper {
   flex: 1;
-  height: var(--p-space-12);
+  height: 44px;
 
-  #{$se23} & {
-    height: 44px;
-
-    @media #{$p-breakpoints-md-down} {
-      height: var(--p-space-12);
-    }
+  @media #{$p-breakpoints-md-down} {
+    height: var(--p-space-12);
   }
 }
 
 .SmallScreenTabsWrapper {
   overflow: hidden;
   padding: var(--p-space-1) var(--p-space-0) var(--p-space-2) var(--p-space-3);
   padding: 0;
-  height: auto;
```

Comment:
This was overriding the `.TabsWrapper` height invocation on mdDown above (`line 36`). 
As this class is only applied at `mdDown`, safe for us to remove this duplicative height rule.